### DISCUSSION TITLE
Full screen keyboard

### DISCUSSION THREAD

**Discussion by hatramatra** [AppGround-io/bluetooth-keyboard-and-mouse-support#129] [Ideas]
Hi, I can't figure out how to use this app as a full screen keyboard for another android device. Something like having one phone in landscape mode as a keyboard for another phone. Is anything like that possible, please? Thanks!

**Comment by AdamKane41** [comment]
Does this look like what you had in mind?
¯——_—_—-==¯——_—_—-==¯——_—_—-==

![Screenshot_2019-11-03-09-45-02](https://user-images.githubusercontent.com/30092603/68087292-5894ca00-fe22-11e9-8ee7-99ae153f8833.png)

¯——_—_—-==¯——_—_—-==¯——_—_—-==¯——_—_—-==
If you're still interested, let me know, it's pretty simple once you understand how to set up the keyboards on both machines..
peace
